src/webportal/src/app/job-submission/utils/utils.js

```diff
@@ -183,6 +182,8 @@ function addTensorBoardCommandsToProtocolTaskRoles(protocol) {
 }
 
 export async function populateProtocolWithDataCli(user, protocol, jobData) {
```

Comment:
Change this function name? It not only inject data cli into the command. It also used to inject the tensorflow cli.

Reply:
done